nomad/rpc.go

```diff
@@ -49,8 +49,9 @@ const (
 
 type rpcHandler struct {
 	*Server
-	logger   log.Logger
-	gologger *golog.Logger
+	logger          log.Logger
+	gologger        *golog.Logger
+	acceptLoopDelay time.Duration
```

Comment:
Define this within `listen()` and pass it to the error handler method unless you put it here for testing.